INTEGRATE_DATA.qmd

```diff
@@ -54,56 +49,45 @@ This enables users of data from different sources to decide which quantities are
 
 11. <https://gdal.org/en/latest/drivers/vector/netcdf.html>
 
-##### Repositories that use this data standard
+### Repositories that use this data standard
 
-1.  NCEI
+1.  [National Centers for Environmental Information (NCEI)](https://www.ncei.noaa.gov/)
 
-2.  NASA EarthData
+2.  [NASA EarthData](https://www.earthdata.nasa.gov/)
 
-### **Topic:** Darwin Core
+## Darwin Core
 
-##### What is it: 
+### What Is It: 
 
-Dublin Core \[Link to Dublin Core section in the primer guidelines.\] is a set of metadata terms used by libraries to describe physical and digital resources. 
+[Dublin Core](https://esipfed.github.io/bds-primer-best-practices/MAKE_YOUR_DATA_INTERNET_READY.html#dublin-core) is a set of metadata terms used by libraries to describe physical and digital resources. 
 
 Darwin Core (DwC) is a glossary of terms…
 
 The Darwin Core archive is a set of interlinked tables…
 
-Darwin Core is a data standard that offers a stable, straightforward and flexible framework for compiling biodiversity data from varied and variable sources. Darwin Core is an extension of Dublin Core for biodiversity informatics. The standard was originally developed by the Biodiversity Information Standards (TDWG, formerly known as the Taxonomic Database Working Group) community, and it is currently maintained by the Darwin Core Maintenance Interest Group; feedback and participation in development is open to the public.. It includes a glossary of terms intended to facilitate the sharing of information about biological diversity by providing identifiers, labels and definitions, improving data reuse in a variety of contexts. In short, it maps information from multiple sources/institutions in a cohesive way for the broader community. Darwin Core is primarily based on taxa, their occurrence in nature as documented by observations, specimens, samples, and related information. Taxonomic occurrence data can be standardized to Darwin Core irrespective of the observing method by which the data were collected (e.g., observational data, genomics, imaging, animal tracking). Through the use of common terminology and controlled vocabularies, downstream users can more easily discover, search, evaluate, integrate and compare datasets\
-\
-
--   Darwin Core Archive

```

Comment:
Removed lines 76 - 80 because it's elaborated on in the Key Information section.